packages/cli/src/commands/proxy/index.ts

```diff
@@ -0,0 +1,91 @@
+import * as http from 'node:http';
+
+import chalk from 'chalk';
+import type { CommandModule } from 'yargs';
+
+import { consoleLog } from '../../utils.js';
+
+import { createOidcResponseHandler, createProxy, isLogtoRequestPath } from './utils.js';
+
+const proxy: CommandModule<
+  unknown,
+  {
+    url?: string;
+    tenant?: string;
+    port: number;
+    endpoint?: string;
+  }
+> = {
+  command: ['proxy'],
+  describe: 'Command for Logto proxy',
+  builder: (yargs) =>
+    yargs
+      .options({
+        url: {
+          alias: ['u', 'sign-in-experience-url'],
+          describe: 'The URL of your custom sign-in experience page',
+          type: 'string',
+        },
+        tenant: {
+          alias: ['t', 'tenant-id'],
+          describe: 'The ID of your Logto Cloud tenant',
+          type: 'string',
+        },
+        port: {
+          alias: 'p',
+          describe: 'The port number where the proxy server will be running on',
+          type: 'number',
+          default: 9000,
+        },
+        endpoint: {
+          alias: 'logto-endpoint',
+          describe:
+            '[Internal] Specify Logto Cloud endpoint URL. E.g. `https://[tenant-id].app.logto.dev` for dev environment. Tenant ID will be omitted when this argument is provided.',
+          type: 'string',
+          hidden: true,
```

Comment:
actually why hide this? we may allow custom domains 😂

Reply:
Oops, forgot about custom domains... OK